python/pyarrow/tests/test_dataset.py

```diff
@@ -1816,7 +1816,7 @@ def test_positional_keywords_raises(tempdir):
 @pytest.mark.parquet
 @pytest.mark.pandas
 def test_read_partition_keys_only(tempdir):
-    BATCH_SIZE = 2 ** 17
+    BATCH_SIZE = 2 ** 15
```

Comment:
Does this still need to be changed? Wouldn't the source break it up?

Reply:
The source breaks it up too much.  I could probably fix the expected number of chunks but this test is easier to understand if it is written write at the splitting boundary.